Makefile

```diff
@@ -68,12 +68,12 @@ vet: ## Run go vet against code.
 	go vet ./...
 
 .PHONY: test-unit
-test-unit: fmt vet
+test-unit: fmt vet ## Run unit tests.
 	go test -v ./internal/... -coverprofile cover.unit.out
 
 .PHONY: test-integration
-test-integration: fmt vet envtest
-	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" go test -v ./test/integration -coverprofile cover.integration.out
+test-integration: fmt vet envtest ## Run integration tests. Use RUN=TestName to run specific tests.
```

Comment:
this comment is helpful though

Reply:
I think I prefer the comment on the line before

Reply:
I tend to agree, this is the convention in our Makefile already. I updated this comment though.